src/components/schedule-builder-view/index.js

```diff
@@ -48,7 +48,8 @@ const getVenuesOptions = (summit) => {
     return venues;
 };
 
-const ScheduleBuilderView = ({summit, scheduleEvents, selectedEvents, currentDay, currentVenue, slotSize, hideBulkSelect, ...props}) => {
+const ScheduleBuilderView = ({summit, scheduleEvents, selectedEvents, currentDay, currentVenue, 
```

Comment:
could be use prop type to define this and prod default values instead ?
much thanks